src/apps/api/views/competitions.py

```diff
@@ -337,6 +337,27 @@ def create_dump(self, request, pk=None):
         serializer = CompetitionCreationTaskStatusSerializer({"status": "Success. Competition dump is being created."})
         return Response(serializer.data, status=201)
 
+    def run_new_task_submissions(self, phase, tasks):
+        tasks_id = set([task.id for task in tasks])
+        submissions = phase.submissions.prefetch_related("children").all()
+
+        for submission in filter(lambda s: s.has_children, submissions):
```

Comment:
```suggestion
        for submission in phase.submissions.filter(has_children=True).prefetch_related("children").all():
```
Is there a reason that this filter doesn't happen using the Django ORM rather than the builtin `filter()`?

Nice prefetch, btw. 😁